test/extended/util/db/mysql.go

```diff
@@ -80,6 +81,11 @@ func (m MySQL) QueryPrivileged(oc *util.CLI, query string) (string, error) {
 			query, masterConf.Env["MYSQL_DATABASE"])).Output()
 }
 
+// QueryPrimary queries the database on primary node as a regular user.
+func (m MySQL) QueryPrimary(oc *util.CLI, query string) (string, error) {
+	return "", errors.New("not implemented")
```

Comment:
I don't like this much... `QueryPrimary` should never be in an interface that MySQL implements.
Specially that in MySQL/PostgreSQL world the naming would be MASTER/SLAVE, right?